.github/workflows/docker-image.yml

```
@@ -47,6 +48,14 @@ jobs:
         with:
           username: ${{ env.DOCKERHUB_USER }}
           password: ${{ env.DOCKERHUB_TOKEN }}
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::${{ secrets.RP_AWS_CRED_ACCOUNT_ID }}:role/${{ vars.RP_AWS_CRED_BASE_ROLE_NAME }}${{ github.event.repository.name }}
+      - name: login to amazon ECR Public
+        uses: aws-actions/amazon-ecr-login@v2
```

```suggestion
      - uses: aws-actions/amazon-ecr-login@v2
```
nit: do not use `name` if it does not add any additional info. the default step name will be `Run aws-actions/amazon-ecr-login@v2`